hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AsyncFSWALProvider.java

```diff
@@ -62,9 +62,9 @@ void init(FileSystem fs, Path path, Configuration c, boolean overwritable, long
       StreamSlowMonitor monitor) throws IOException, CommonFSUtils.StreamLacksCapabilityException;
   }
 
-  private EventLoopGroup eventLoopGroup;
```

Comment:
Add comments to say this will be used in tests.

Reply:
@Apache9 ，ok ,fixed.